### ISSUE TITLE
Accessibility: Skip Navigation not Present

### ISSUE THREAD

**Issue by michaelspellacy** [alistapart/AListApart#413]
In looking for examples of skip navigation links, I noticed that ALA seemed to be missing one and seems to be combined with your mobile menu button. It really should be present when tabbing, for low-vision folks who may also be using assistive tech, etc. Probably best to keep your menu button and skip link separated, too.

`<a class="go-to-nav" href="#main-nav" title="skip to nav">Menu</a>`

**Comment by awkawk** [comment]
@michaelspellacy I've recommended against a skip link in the past, and still do. There are plugins available to enable keyboard access to semantic structure in a page, and assistive technologies such as screen readers also enable users to navigate past the topnav.

For the mobile version, the next tab stop after the hamburger menu is the main heading, so a skip nav link would actually require more effort.

**Comment by michaelspellacy** [comment]
@awkawk  That's cool, Probably an edge case, but I also find myself thinking about it from POV of user who may suddenly find themselves disabled or unable to use a pointing device and who would not not be privy to AT at that point . Trust your insight, of course, so if you wish to close this, then that is fine.

**Comment by awkawk** [comment]
@michaelspellacy I might feel differently if ALA had a massive megamenu but even for the full site there are only a handful of tab stops before the actual content.

I'm not big on adding features to web sites that should be handled by the browser or OS. When the browser/OS completely drops the ball I will likely feel otherwise, but things like text resizing widgets or alternative style sheet pickers as a way to make a site more accessible tend to only provide fractional support for users and play a part in convincing browser and OS vendors that they don't need to do anything because the authors can (but usually don't) handle it.

**Comment by michaelspellacy** [comment]
I agree.
